dev/manual_tests/lib/material_arc.dart

```diff
@@ -210,7 +210,7 @@ class _PointDemoState extends State<_PointDemo> {
               child: Text(
                 'Tap the refresh button to run the animation. Drag the green '
                 "and red points to change the animation's path.",
-                style: Theme.of(context).textTheme.caption.copyWith(fontSize: 16.0),
+                style: Theme.of(context).textTheme.caption?.copyWith(fontSize: 16.0),
```

Comment:
flutter is using ! in TextTheme since they can never be null (one day the old one is going to be deprecated and this is going to disappear).